azure-mgmt-datafactory/azure/mgmt/datafactory/models/__init__.py

```diff
@@ -59,8 +59,6 @@
 from .operation_list_response import OperationListResponse
 from .azure_data_lake_analytics_linked_service import AzureDataLakeAnalyticsLinkedService
 from .hd_insight_on_demand_linked_service import HDInsightOnDemandLinkedService
-from .netezza_linked_service import NetezzaLinkedService
```

Comment:
Why is this removed?

Reply:
Because @haricr fork is not up to date to latest master, and we build from the branch, not from the state that would be the PR after merge.

See, 18 commits behind:
![image](https://user-images.githubusercontent.com/1050156/36283214-a36d9da2-1258-11e8-8da2-657285748a8d.png)

Reply:
That's an interesting scenario indeed for the automation system.

Reply:
Either way, we need to block the merge of this PR until we release 0.5.0. So please do not merge this.

Reply:
@hvermis two things:
- I'm working right now to improve the system and play with git to be smarter
- The base of this PR is `restapi_auto_datafactory/resource-manager`, not `master`. I added a layer of branches. This PR will be merged automatically when the Swagger will be merged. The one you don't want to merge too early is the cumulative PR that contains all your work in progress:
https://github.com/Azure/azure-sdk-for-python/pull/1972

Am I clear?

Reply:
I have approved #1972 and we want that one merged, but not the latest changes in this one that have only Hari's update - Filteractivity, parameter in Linked service etc.

Reply:
I you don't merge a Swagger that you don't want to release we're good :)
That's the process anyway, since merging a Swagger means ready to release for other languages.